src/devices/video/nereid.h

```diff
@@ -0,0 +1,43 @@
+// license:BSD-3-Clause
+// copyright-holders:Sven Schnelle
+#ifndef MAME_VIDEO_NEREID_H_
+#define MAME_VIDEO_NEREID_H_
```

Comment:
Please follow the convention for `#include` guards - we don't use a trailing underscore.

Reply:
fixed, thanks